.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we be committing the .swiftpm dir? I don't think there's a consensus on this, but I'd rather not commit unless we can prove it's necessary.

Reply:
Yes, this might have created  due to my testing of spm. Removed.